ethdb/pebble/pebble.go

```diff
@@ -187,10 +187,18 @@ func New(file string, cache int, handles int, namespace string, readonly bool, e
 		"handles", handles, "memory table", common.StorageSize(memTableSize))
 
 	db := &Database{
-		fn:           file,
-		log:          logger,
-		quitChan:     make(chan chan error),
-		writeOptions: pebble.Sync,
+		fn:       file,
+		log:      logger,
+		quitChan: make(chan chan error),
+
+		// Use asynchronous write mode by default. Otherwise, the overhead of frequent fsync
+		// operations can be significant, especially on platforms with slow fsync performance
+		// (e.g., macOS) or less capable SSDs.
+		//
+		// Note that enabling async writes means recent data may be lost in the event of an
+		// application-level panic (writes will also be lost on a machine-level failure,
+		// of course). Geth is expected to handle recovery from an unclean shutdown.
+		writeOptions: pebble.NoSync,
```

Comment:
writeOptions: pebble.Sync--> pebble.NoSync !!!

1. do more test
2. performance may be improved.